build/android/patches/irrlicht-back_button.patch

```diff
@@ -1,14 +1,6 @@
---- irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp	2014-06-03 20:56:21.289559503 +0200
-+++ irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp.orig	2014-06-03 20:57:39.281556749 +0200
-@@ -423,6 +423,7 @@
- 			}
- 			
- 			device->postEventFromUser(event);
-+			status = 1;

```

Comment:
Removing this is wrong as well as keeping it. status must be 1 if postEventFromUser returns true, otherwise it must be false. That's at least what I think is the most proper code here.